lib/data_magic.rb

```diff
@@ -76,12 +76,20 @@ def self.search(terms, options = {})
       body: query_body
     }
 
+    # Per https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-body.html:
+    # "the search_type and the query_cache must be passed as query-string parameters"
+    if options[:command] == 'stats'
+      full_query.merge! :search_type => 'count'
+    end
+
     logger.info "FULL_QUERY: #{full_query.inspect}"
 
     time_start = Time.now.to_f
     result = client.search full_query
+
     search_time = Time.now.to_f - time_start
-    logger.info "ES query time (ms): #{result["took"]} ; Query fetch time (s): #{search_time} ; result: #{result.inspect[0..500]}"
+    logger.info "ES query time (ms): #{result["took"]} ; Query fetch time (s): #{search_time} ; result: #{result.inspect}"
```

Comment:
The `[0..500]` is there because some results were massive and filling the logs. I'd prefer to keep it in by default. (Actually, what I'd _really_ prefer is that we substantially change how we do logging and debugging, but that's for another time)